src/Common/src/Interop/Unix/libssl/Interop.OpenSsl.cs

```diff
@@ -371,7 +355,7 @@ private static void Disconnect(SafeSslHandle context)
             if (retVal < 0)
             {
                 //TODO (Issue #3362) check this error
-                libssl.SSL_get_error(context, retVal);
+                Crypto.SslGetError(context, retVal);
```

Comment:
We're ignoring the result of this SslGetError call.  The TODO comment refers to issue 3362, which is closed.  3362 refers to issue 3954 as handling the rest, but 3954 appears to be specifically about channel binding.